website/i18n/zh/code.json

```diff
@@ -30,5 +34,221 @@
   "docs.webpage.title.DocumentSubtitle": {
     "message": "让产品与技术背后的细节更加可视化",
     "description": "We love open source."
+  },
+  "team.webpage.title.Team": {
+    "message": "团队介绍",
+    "description": "Team"
+  },
+  "team.webpage.title.DocumentSubtitle": {
+    "message": "We love open source.",
+    "description": "We love open source."
+  },
+  "team.webpage.content.Contributors": {
+    "message": "贡献者列表",
+    "description": "Contributors"
+  },
+  "team.webpage.content.ContributorsSub": {
+    "message": "你可以从 GitHub 的贡献记录中查看所有 Apache APISIX 的贡献者列表",
+    "description": "You can find all contributors of Apache APISIX from GitHub contribution list."
+  },
+  "team.webpage.content.ContributorOverTime": {
+    "message": "贡献者变化趋势图",
+    "description": "Contributor Over Time"
+  },
+  "team.webpage.content.ContributorOverTimeNote": {
+    "message": "注：下图中的贡献者数量包含了 Apache APISIX 所有仓库的贡献者",
+    "description": "Note: This graph contains contributors from all repos under Apache APISIX"
+  },
+  "team.webpage.content.BecomeACommitter": {
+    "message": "成为 Apache APISIX Committer",
+    "description": "Become A Committer"
+  },
+  "team.webpage.content.BecomeACommitterInto": {
+    "message": "Apache APISIX 社区遵循并使用 Apache 社区提交规范，在贡献者积极参与 APISIX 社区后，项目管理委员会（PMC）将决定是否邀请贡献者加入 Committers 和 PMC。",
+    "description": "The Apache APISIX community follows the Apache Community process on accepting a new committer. After contributors actively participate in the APISIX community, PMC and Committers will decide to invite contributors to join Committers and PMC."
+  },
+  "team.webpage.content.StartContributeBtn": {
+    "message": "开始参与社区贡献",
+    "description": "Start Contribute"
+  },
+  "architecture.component.title.name": {
+    "message": "适用于超大规模、复杂的业务系统",
+    "description": "Building for large-scale, high value systems"
+  },
+  "architecture.component.title.subtitle": {
+    "message": "Apache APISIX 作为云原生架构的开源 API 网关，可以为海量 API 和微服务提供安全可靠的高性能、动态、可扩展的管理平台。",
+    "description": "Apache APISIX lets you build Cloud-Native Microservices API gateways, delivering the ultimate performance, security, open source and scalable platform for all your APIs and microservices."
+  },
+  "architecture.component.card.caption": {
+    "message": "Apache APISIX 基于 Nginx 与 etcd，相较于传统的 API 网关，它提供了动态路由、插件热加载等诸多能力",
+    "description": "Apache APISIX is based on Nginx and etcd. Compared with traditional API gateways, APISIX has dynamic routing and hot-loading plugins"
```

Comment:
```suggestion
    "message": "Apache APISIX 基于 NGINX 与 etcd，相较于传统的 API 网关，它提供了动态路由、插件热加载等诸多能力",
    "description": "Apache APISIX is based on NGINX and etcd. Compared with traditional API gateways, APISIX has dynamic routing and hot-loading plugins"
```

Reply:
You can see https://www.nginx.com/, NGINX(not Nginx)is used wherever NGINX-related content mentioned on the official website